package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack serve --open",
```

Comment:
- [ ] **Optional:** You could also add the `dev` script as follows `"dev": "webpack --mode development",`